docs/theme_architecture/blocks/schema/variables/variable_types/package.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: Package
+parent: Variable types
+---
+
+# Package
+
+Renders a select input from which a Creator can select a package from the current companies **listable** packages.
+> Listable packages are packages that are not in a draft state.
+
+Returns a [Package]({% link docs/reference/objects/package.md %}) object.
```

Comment:
This link doesn't render correctly

Reply:
<img width="630" alt="image" src="https://user-images.githubusercontent.com/13543612/234540570-900558cb-3c4f-4c25-95b9-c60e0f6664c6.png">
What do you see?

Looks fine to me 👀 ?